plugins/GSdx/GSDeviceOGL.cpp

```diff
@@ -571,10 +571,10 @@ GLuint GSDeviceOGL::CreateSampler(bool bilinear, bool tau, bool tav)
 
 	gl_SamplerParameterf(sampler, GL_TEXTURE_MIN_LOD, 0);
 	gl_SamplerParameterf(sampler, GL_TEXTURE_MAX_LOD, 6);
+	int anisotropy = theApp.GetConfig("MaxAnisotropy", 0);
 
-	if (GLLoader::found_GL_EXT_texture_filter_anisotropic && !!theApp.GetConfig("AnisotropicFiltering", 0) && !theApp.GetConfig("paltex", 0)) {
-		int anisotropy = theApp.GetConfig("MaxAnisotropy", 1);
-		if (anisotropy > 1) // 1 is the default in opengl so don't do anything
+	if (GLLoader::found_GL_EXT_texture_filter_anisotropic && anisotropy && !theApp.GetConfig("paltex", 0)) 
```

Comment:
Whitespace error on this line. Also, the blank line should be before int anisotropy, not after. For style consistency with the rest of the file the '{' and '}' should be removed.